paimon-core/src/main/java/org/apache/paimon/utils/TagManager.java

```diff
@@ -235,8 +237,21 @@ private void doClean(
         }
 
         // delete manifests
-        tagDeletion.cleanUnusedManifests(
-                taggedSnapshot, tagDeletion.manifestSkippingSet(skippedSnapshots));
+        success = true;
+        Set<String> manifestSkippingSet = new HashSet<>();
```

Comment:
Variable 'manifestSkippingSet' should been initialized to null.

Reply:
ok，done